index.html

```diff
@@ -28,30 +36,53 @@
 
         <!-- you can nest rows inside of columns! -->
         <!-- our board has 3 rows, each with 3 columns inside -->
+
+
+        <p class="turn">Player X goes first...</p>
+        
+        <div class="container" id="roundbox">
+
+
+        <div class="masterbox">
         <div class="row">
-          <div class="col-xs-4 box"></div>
-          <div class="col-xs-4 box"></div>
-          <div class="col-xs-4 box"></div>
+          <div class="col-xs-4 box square" id="a"></div>
+          <div class="col-xs-4 box square" id="b"></div>
+          <div class="col-xs-4 box square" id="c"></div>
         </div>
 
         <div class="row">
-          <div class="col-xs-4 box"></div>
-          <div class="col-xs-4 box"></div>
-          <div class="col-xs-4 box"></div>
+          <div class="col-xs-4 box square" id="d"><p></p></div>
+          <div class="col-xs-4 box square" id="e"></div>
+          <div class="col-xs-4 box square" id="f"></div>
         </div>
 
         <div class="row">
-          <div class="col-xs-4 box"></div>
-          <div class="col-xs-4 box"></div>
-          <div class="col-xs-4 box"></div>
+          <div class="col-xs-4 box square" id="g"></div>
+          <div class="col-xs-4 box square" id="h"></div>
+          <div class="col-xs-4 box square" id="i"></div>
         </div>
       </div>
     </div>
   </div>
+  </div>
+  </div>
+
+    
+      <div class="button">
+
+          <input class="btn btn-default" type="submit" value="Reset">
   
+      </div>
+    </div>
+
+    <br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/>
   <!-- jquery -->
+  <!-- jquery -->
+  
+  <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.0.0-alpha1/jquery.min.js"></script>
```

Comment:
Careful, jQuery 3 is in alpha. We're typically using jQuery 1.10 or 1.11 so be cautious that sometimes stuff in jQuery 3 will not work.